### ISSUE TITLE
SuperVisor Cannot open an HTTP server:

### ISSUE THREAD

**Issue by Murad-Shukurlu** [Supervisor/supervisor#1183]
Hi friends ,

I have problem on Ubuntu server 16.04 for Openedx so  after reboot my server down .

I found that SuperVisor is not active . I run this command : 

`sudo service supervisor start`

then output is : 

Job for supervisor.service failed because the control process exited with error code. See "systemctl status supervisor.service" and "journalctl -xe" for details.
then i run systemctl status supervisor.service 

```
● supervisor.service - supervisord - Supervisor process control system
   Loaded: loaded (/etc/systemd/system/supervisor.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Sat 2018-12-15 19:25:04 UTC; 1min 18s ago
     Docs: http://supervisord.org
  Process: 27723 ExecStart=/edx/app/supervisor/venvs/supervisor/bin/supervisord --configuration /edx/app/supervisor/supervisord.conf (code=exited, status=2)

Dec 15 19:25:04 ip-172-31-14-220 systemd[1]: Starting supervisord - Supervisor process control system...
Dec 15 19:25:04 ip-172-31-14-220 supervisord[27723]: Error: Cannot open an HTTP server: socket.error reported errno.EACCES (13)
Dec 15 19:25:04 ip-172-31-14-220 supervisord[27723]: For help, use /edx/app/supervisor/venvs/supervisor/bin/supervisord -h
Dec 15 19:25:04 ip-172-31-14-220 systemd[1]: supervisor.service: Control process exited, code=exited status=2
Dec 15 19:25:04 ip-172-31-14-220 systemd[1]: Failed to start supervisord - Supervisor process control system.
Dec 15 19:25:04 ip-172-31-14-220 systemd[1]: supervisor.service: Unit entered failed state.
Dec 15 19:25:04 ip-172-31-14-220 systemd[1]: supervisor.service: Failed with result 'exit-code'.
```
i can not fix this : 

My supervisord.conf file : 

```
; supervisor config file

[unix_http_server]
file=/edx/var/supervisor/supervisor.sock   ; (the path to the socket file)
chmod=0700                       ; sockef file mode (default 0700)

[supervisord]
logfile=/edx/var/log/supervisor/supervisord.log ; (main log file;default $CWD/supervisord.log)
pidfile=/edx/var/supervisor/supervisord.pid ; (supervisord pidfile;default supervisord.pid)
childlogdir=/edx/var/log/supervisor            ; ('AUTO' child log dir, default $TEMP)

; the below section must remain in the config file for RPC
; (supervisorctl/web interface) to work, additional interfaces may be
; added by defining them in separate rpcinterface: sections
[rpcinterface:supervisor]
supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface

[supervisorctl]
serverurl=unix:///edx/var/supervisor/supervisor.sock ; use a unix:// URL  for a unix socket

; The [include] section can just contain the "files" setting.  This
; setting can list multiple files (separated by whitespace or
; newlines).  It can also contain wildcards.  The filenames are
; interpreted as relative to this file.  Included files *cannot*
; include files themselves.

[inet_http_server]
port = 127.0.0.1:9001

[include]
files = /edx/app/supervisor/conf.d/*.conf
```

Even i changed chmod to 766 but no use .
When i change chmod to 777 i get an error .

unix:///edx/var/supervisor/supervisor.sock refused connection
 
How can i handle those of all ?

* My machine on AWS . My server down now this happened after reboot of my server .

**Comment by mnaberez** [comment]
```
Dec 15 19:25:04 ip-172-31-14-220 supervisord[27723]: Error: Cannot open an HTTP server: socket.error reported errno.EACCES (13)
```

Based on the message above, it may be permissions related.  Please ask questions about installing or using Supervisor on the [supervisor-users mailing list](https://lists.supervisord.org/mailman/listinfo/supervisor-users).  There may be users there who can help troubleshoot.  We use this issue tracker mainly for bugs in Supervisor itself.

**Comment by Murad-Shukurlu** [comment]
> ```
> Dec 15 19:25:04 ip-172-31-14-220 supervisord[27723]: Error: Cannot open an HTTP server: socket.error reported errno.EACCES (13)
> ```
> Based on the message above, it may be permissions related. Please ask questions about installing or using Supervisor on the [supervisor-users mailing list](https://lists.supervisord.org/mailman/listinfo/supervisor-users). There may be users there who can help troubleshoot. We use this issue tracker mainly for bugs in Supervisor itself.

@mnaberez Thank you for sincerly answer . But the mailing list doesnot work .

**Comment by mnaberez** [comment]
> But the mailing list doesnot work .

There is an open issue to fix the certificate warning in #947.

**Comment by zord1k** [comment]
I fixed this issue by editing /etc/systemd/system/multi-user.target.wants/supervisord.service
I changed RuntimeDirectoryMode to 777

/etc/supervisord.conf
[unix_http_server]
file=/run/supervisor/supervisor.sock

[supervisord]
pidfile=/run/supervisor/supervisord.pid

[supervisorctl]
serverurl=unix:///run/supervisor/supervisor.sock
